Exercise_1_TF_env/Exercise_1.txt

```diff
@@ -1,28 +1,34 @@
 # Request GPU partition:
+#####Use after training##############
 srun --partition=gpu --nodes=1 –-cpus-per-task=1 --pty --gres=gpu:p100:1 --mem=10G --time=02:00:00 /bin/bash
+#####################################
 
-# Build tensorflow env from scratch in your $HOME directory:
-module load anaconda3/2022.05
-module load cuda/11.2
-conda create --name TF_env python=3.9 -y #where TF_env is the name of the conda environment
-source activate TF_env #load the virtual conda environment "TF_env"
-export LD_LIBRARY_PATH=$HOME/.conda/envs/TF_env/lib:$LD_LIBRARY_PATH
-conda install -c conda-forge cudatoolkit=11.2 cudnn=8.1.0 -y
-pip install tensorflow
-
+#####Use during training###############
+srun --partition=reservation --reservation=bootcamp_gpu_2023 --gres=gpu:1 --nodes=1 --cpus-per-task=1 --pty --mem=10G --time=02:00:00 /bin/bash 
+#######################################
 
-## Loading the pre-existing tf_env environment
-module load cuda/11.1 
-module load anaconda3/2021.05
+# Build tensorflow env from scratch in your $HOME directory:
+srun --partition=gpu --nodes=1 –-cpus-per-task=1 --pty --gres=gpu:p100:1 --mem=10G --time=02:00:00 /bin/bash
+module load anaconda3/2022.05 cuda/11.2
```

Comment:
We need to change this to cuda/11.8 and do away with that old system-wide environment. Same for Pytorch. The revised material on RTD shows the updated version for PyTorch but needs to be updated for TF.

Reply:
@manasvita - I updated the instructions and have tested this. It works but it appears to be dependent on installation order to get JupyterLab to work well with it. I also write a sbatch job to install and create the environment which might be better for the workshop. Have the attendees submit the job in the beginning while we walk through the slides the environment is being built in the background for all of them. 

I am going to update the slides next but I wanted to get all of this code tested again using the new environment. I just used the installation instructions from TensorFlow plus upgrading pip (out of routine) and install JupyterLab to get it working on OOD. 

Thank you!!